andino_navigation/README.MD

```diff
@@ -0,0 +1,45 @@
+# Nav2 with Andino robot
+
```

Comment:
Can you add a small description here? (super simple one) and then add a `_Note: Nav2 is only configured to work with the simulation for the moment. Support to real robot is forthcoming._`